### PR TITLE
feat: add lightweight styling resources to RatingControl

### PR DESCRIPTION
**Pull request by Arieldelossantos**
﻿GitHub Issue: https://github.com/unoplatform/uno.themes-private/issues/14

## PR Type

What kind of change does this PR introduce?

- Feature

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested the changes where applicable:
	- [ ] UWP
	- [ ] iOS
	- [ ] Android
	- [ ] WASM
	- [ ] MacOS
- [ ] Updated the documentation as needed:
	- [ ] [General Doc Update](https://github.com/unoplatform/Uno.Themes/tree/master/doc)
	- [ ] [material-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-controls-styles.md)
	- [ ] [cupertino-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/cupertino-controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/Uno.Themes/blob/master/doc/lightweight-styling.md)
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Themes.WinUI.Markup/Theme/RatingControl.cs [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+				public static class Secundary
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "SecundaryRatingControlForeground")]
+						public static ResourceValue<Brush> Default => new("RatingControlForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "SecundaryRatingControlForegroundUnselected")]
+						public static ResourceValue<Brush> Unselected => new("RatingControlForegroundUnselected", true);
```
Comment (ramezgerges) [inline src/library/Uno.Themes.WinUI.Markup/Theme/RatingControl.cs:79]:
```suggestion
						public static ResourceValue<Brush> Unselected => new("SecondaryRatingControlForegroundUnselected", true);
```

And the same for everything in `Secondary`

File: src/library/Uno.Themes.WinUI.Markup/Theme/RatingControl.cs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+				}
+
+				public static class Secundary
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "SecundaryRatingControlForeground")]
+						public static ResourceValue<Brush> Default => new("RatingControlForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "SecundaryRatingControlForegroundUnselected")]
```
Comment (ramezgerges) [inline src/library/Uno.Themes.WinUI.Markup/Theme/RatingControl.cs:78]:
```suggestion
						[ResourceKeyDefinition(typeof(Brush), "SecondaryRatingControlForegroundUnselected")]
```

And the same for most of `Secondary`

**Review by ramezgerges — CHANGES_REQUESTED** [review]

File: src/library/Uno.Themes.WinUI.Markup/Theme/RatingControl.cs [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlPlaceholderForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("RatingControlPlaceholderForegroundPointerOver", true);
+					}
+
+					public static class CaptionForeground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlCaptionForeground")]
+						public static ResourceValue<Brush> Default => new("RatingControlCaptionForeground", true);
+					}
```
Comment (ramezgerges) [inline src/library/Uno.Themes.WinUI.Markup/Theme/RatingControl.cs:50]: This doesn't need a class.
```suggestion
						[ResourceKeyDefinition(typeof(Brush), "RatingControlCaptionForeground")]
						public static ResourceValue<Brush> CaptionForeground => new("RatingControlCaptionForeground", true);
```

File: src/library/Uno.Themes.WinUI.Markup/Theme/RatingControl.cs [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+					[ResourceKeyDefinition(typeof(double), "RatingControlHeight")]
+					public static ResourceValue<double> Height => new("RatingControlHeight", true);
+
+					[ResourceKeyDefinition(typeof(double), "RatingControlCaptionHeight")]
+					public static ResourceValue<double> CaptionHeight => new("RatingControlCaptionHeight", true);
+				}
+
+				public static class Secundary
```
Comment (kazo0) [inline src/library/Uno.Themes.WinUI.Markup/Theme/RatingControl.cs:71]:
```suggestion
				public static class Secondary
```

File: src/library/Uno.Material/Styles/Controls/v2/RatingControl.xaml [hunk 5]
```diff
@@ -1,272 +1,271 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```
Comment (kazo0) [inline src/library/Uno.Material/Styles/Controls/v2/RatingControl.xaml:1]: Make sure to update the docs of the lightweight styling keys for RatingControl

**Review by kazo0 — APPROVED** [review]

**Pull request merged**